### PR TITLE
EWL-4426: Visual clean up of topic, press release and news page from UX collab session

### PR DESCRIPTION
**Pull request by designerneil**
## Ticket(s)

**Jira Ticket**
- [EWL-4426: SG2 | Visual cleanup](https://issues.ama-assn.org/browse/EWL-4426)

## Description
Removes extraneous p tag in the news.json which caused vertical rhythm issues
Removes paragraph for topics page article stub so it matches the zeplin comp 
Changes article metadata to use small type 
Adds bottom margin to membership blocks preventing them touching
Updates two column right template grid sizing. UX has asked that the right sidebar be smaller

## To Test
- [ ] `gulp serve`
- [ ] Go to [news page](http://localhost:3000/?p=pages-news)
- [ ] Observe the main body copy is vertically lined up with the tools list in the right sidebar
- [ ] Observe the right sidebar width reduced and the main body enlarged 

- [ ] Go to [press release page](http://localhost:3000/?p=pages-press-release)
- [ ] Observe the main body copy is vertically lined up with the tools list in the right sidebar
- [ ] Observe the right sidebar width reduced and the main body enlarged 

- [ ] Go to [topic page](http://localhost:3000/?p=pages-topic)
- [ ] Observe the article stub block under "Example Article using the "Hero" No Media mode" only has an image and title
- [ ] Observe the membership blocks in the right sidebar not touching anymore

## Visual Regressions
run `gulp backstop test`

## Relevant Screenshots/GIFs
screen shot 2018-02-15 at 2 00 25 pm
screen shot 2018-02-15 at 2 00 16 pm
screen shot 2018-02-15 at 2 00 08 pm

## Remaining Tasks
N/A

## Additional Notes
N/A
---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

**Comment by nstriedinger** [comment]
Is this supposed to line up?

screen shot 2018-02-16 at 11 34 35 am

**Comment by nstriedinger** [comment]
This looks odd-- is this right?

screen shot 2018-02-16 at 11 41 26 am

File: styleguide/source/assets/scss/02-molecules/_article-metadata.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
-.ama__article-metadata {
+.ama__article-stub__metadata {
```
Comment (nstriedinger) [inline styleguide/source/assets/scss/02-molecules/_article-metadata.scss:1]: -- by being so specific are you saying that different Articles have different styled Meta data? If so, you need to change the file name too.

**Comment by designerneil** [comment]
@nstriedinger both items being not perfectly centered has to do with the line heights in the font. I'm going ask UX if they are cool with it being slightly off. Otherwise we will need to override the default line heights for the h3 on the topic page and h4 on the press release page for it to line up.

**Review by nstriedinger — CHANGES_REQUESTED** [review]

**Comment by designerneil** [comment]
@nstriedinger UX is fine with the type being slightly off.

File: styleguide/source/assets/scss/02-molecules/_article-metadata.scss (hunk 1 above)
Comment (designerneil) [reply styleguide/source/assets/scss/02-molecules/_article-metadata.scss:1]: I think what I'm saying here is metadata is a child of article stub

**Review by nstriedinger — APPROVED** [review]

**Pull request merged**